rts/Game/Game.cpp

```diff
@@ -1402,7 +1405,9 @@ bool CGame::UpdateUnsynced(const spring_time currentTime)
 		unitTracker.SetCam();
 
 	camera->Update();
+	CamVisibleQuads.Update();
 	shadowHandler.Update();
+	CamVisibleShadowQuads.Update();
```

Comment:
Wouldn't it be better if instead putting the `Update()` function in the right place I could just type:
`readMap->GridVisibility(CCameraHandler::GetCamera(CCamera::CAMTYPE_SHADOW), &drawer, 1e9, HM_SQUARE_SIZE);`
but this currently isn't supported